### ISSUE TITLE
check basic authentication

### ISSUE THREAD

**Issue by marcosh** [mvlabs/ze-workshop#4]

**Comment by maraspin** [comment]
Can't remember... Did we decide a specific way to implement basic auth (IE existing piece of middleware)? Tnx

**Comment by marcosh** [comment]
yes, it's already implemented. Look in `config/routes.php` to start
